pallets/msa/Cargo.toml

```diff
@@ -13,32 +13,31 @@ version = "0.0.0"
 targets = ["x86_64-unknown-linux-gnu"]
 
 [dependencies]
-log = "0.4.17"
-codec = { package = "parity-scale-codec", version = "3.6.1", default-features = false, features = [
+log = { workspace = true }
+parity-scale-codec = { workspace = true, features = [
```

Comment:
I don't quite understand the `codec` vs `parity-scale-codec` swap. Don't we want the generic codec one?

Reply:
Ahh forgot to put it back.